team/index.html

```diff
@@ -1560,7 +1560,21 @@ <h1>Enablement Team</h1>
 						<span>Event and Program Manager</span>
 					</div>
 				</div>
-
+				<div class="col-md-3 col-sm-6 speaker-column">
+				<div class="speaker">
+					<div class="image-holder">
+						<img class="background-image" alt="Chung Wei Tat" src="../img/team/ChungWeiTat.jpg">
+						<div class="hover-state text-center preserve3d">
+							<div class="social-links vertical-align">
+								<a href="https://twitter.com/cweitat"><i class="icon social_twitter"></i></a>
```

Comment:
Open page in a new tab. With target = blank

Reply:
Now updated to open in new tab.